src/pages/zh/dev-tools/cli/bubblegum/index.md

```
@@ -0,0 +1,92 @@
+description: 使用 Bubblegum V2 程序创建和管理压缩 NFT
+---
+
+# Bubblegum（压缩 NFT）
+
+{% callout type="note" %}
+这些 CLI 命令仅适用于 **Bubblegum V2**。Bubblegum V2 使用 [Metaplex Core 集合](/smart-contracts/core/collections)，与 Bubblegum V1 树或 Token Metadata 集合不兼容。
+{% /callout %}
+
+Bubblegum 是 Metaplex 的压缩 NFT（cNFT）程序，允许您以传统 NFT 成本的一小部分创建 NFT。通过使用并发默克尔树进行状态压缩，在初始树创建成本之后，压缩 NFT 只需支付交易成本即可铸造。
+
+## 核心概念
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)